### ISSUE TITLE
A possible solution for Music at CampJS 2014-1

### ISSUE THREAD

**Issue by geoffreyd** [campjs/campjs-iii#5]
This looks like a pretty neat service ... we might have to just make sure it can run offline.

https://github.com/play/play

**Comment by sporto** [comment]
Whatever we do, we need music that pleases mostly everyone or at least people don't dislike it, we had several periods on previous camps where a few people will dictate music that was hardly enjoyable, not because I am developer I like 80's video games music as background

**Comment by timoxley** [comment]
this is a tricky one. I'd prefer to just pick somebody to be in charge of queuing music and be done with it. Not a big fan of allowing anyone to pick music as then you just get everyone showing off their best novelty songs.

There's a bit of an art to picking the right music, I don't claim to be good at this at all. Perhaps somebody in the campjs community has some DJ skills and could help out here.

**Comment by jonathanong** [comment]
karaoke

**Comment by antulik** [comment]
agree with @timoxley we had the same problem at jelly and I'd say we are better off with one or two people in charge of playlist

**Comment by rvagg** [comment]
Right, I'm going to be assertive here. I'm in charge of music since I'm general soundie for the weekend. Happy to relinquish if someone else comes forward with a better claim to this job.

My choice of music will be similar to the previous two CampJS' playlists, general downtempo/chill/electronic working music. My playlist will likely consist largely of: Emancipator, Bonobo, Saltillo, Little People, Nomak, Tor, Beats Antique, Berry Weight, Bitter:Sweet, Late Night Alumni. Happy to take suggestions on music with a similar feel that's not too distracting for people who mainly want background noise. Also happy to hear any feedback if people had problems with the kind of music we've had at the previous events.

**Comment by timoxley** [comment]
@rvagg :+1: :+1:  sold. 

> Right, I'm going to be assertive here. I'm in charge of music…

Thanks Rod. This is the right attitude, I'd like others to feel welcome to nominate themselves to be in charge of any other aspect of campjs of which they feel they could provide a valuable contribution.

**Comment by markmandel** [comment]
Just thought I'd throw a random fun thought in. Could also set up a http://plug.dj/ room and use that as the music source.

Of course it uses up a decent amount of bandwidth, so that may make it a no go. Just an idea.

**Comment by timoxley** [comment]
@markmandel yeah bandwidth will be prohibitive, solution needs to run locally.

**Comment by markmandel** [comment]
@timoxley That sounds an awful lot like a challenge :smile:

**Comment by juliangruber** [comment]
@hij1nx we could bring our guitars and play through amp.js, maybe someone wants to do substackesque algorithmic music too?

**Comment by heapwolf** [comment]
Hah! that would be awesome, but i have several stops afterword so i wont be able to bring something that big. :(

**Comment by juliangruber** [comment]
ok, yeah i'd worried about my guitar too even for that one stop

**Comment by timoxley** [comment]
@juliangruber @hij1nx I'd love this to happen. Let's see if we can find someone who can bring some guitars

**Comment by MauriceButler** [comment]
I have a less than awesome one but it is electric acoustic with USB out.

I cant play it yet though would love someone to teach me!

**Comment by juliangruber** [comment]
any time haha
